.github/CODEOWNERS

```diff
@@ -743,7 +743,7 @@ packages/kbn-rule-data-utils @elastic/security-detections-response @elastic/resp
 x-pack/plugins/rule_registry @elastic/response-ops @elastic/obs-ux-management-team
 x-pack/plugins/runtime_fields @elastic/kibana-management
 packages/kbn-safer-lodash-set @elastic/kibana-security
-x-pack/test/security_api_integration/plugins/saml_provider @elastic/kibana-security
+packages/kbn-saml-provider-plugin @elastic/kibana-security
```

Comment:
Why not leave this to the jsonc file instead?  ..and let the owners file be generated from the jsonc file

Reply:
good point, removing

Reply:
I thought we were removing this line?  No?

Reply:
it is auto-added by kibanamachine :)